_reports/ishanSrt.md

```diff
@@ -10,11 +10,11 @@ organisation_link: https://coala.io
 report_year: 2018
 project: Quickstart Green Mode
 project_link: https://summerofcode.withgoogle.com/projects/#5792652830703616
-tarball: https://cdn.rawgit.com/ishanSrt/ishanSrt.github.io/d859b05e/ishanSrt.tar.gz
-sha256sum: 4dcee0f017d832965ca44a76d0a8b20f92d68fd2fb820975984a51b4ff34806a
+tarball: https://cdn.rawgit.com/ishanSrt/ishanSrt.github.io/448110a9/ishanSrt.tar.gz
```

Comment:
You have a lot of `._*` files in here, and `.DS_Store`

Any chance you can remove them?

Otherwise looks good.